sdk/storage/Azure.Storage.Blobs/tests/ContainerClientTests.cs

```diff
@@ -52,6 +52,17 @@ public void Ctor_ConnectionString()
 
         [Test]
         public async Task Ctor_ConnectionString_Sas()
+        {
+            await Perform_Ctor_ConnectionString_Sas();
+        }
+
+        [Test]
+        public async Task Ctor_ConnectionString_Sas_Include_Table()
+        {
+            await Perform_Ctor_ConnectionString_Sas(includeTable: true);
+        }
+
+        private async Task Perform_Ctor_ConnectionString_Sas(bool includeTable = false)
```

Comment:
Nit - you could use TestData or whatever nUnit calls it to pass these values in instead of wiring up separate tests.

Reply:
Updated